bin/_create_deployment.bat

```diff
@@ -0,0 +1,7 @@
+REM Download and extract Python
+REM Download and extract libs
+REM  sqlite
+REM  mod_spatialite
+REM Copy new pth file
```

Comment:
@robintw  - in the longer term we'll also have to distribute enough Postgis dependencies to  make a client connection to a networked PG database.  I _briefly_ thought  the *requirements.txt* would have enough, but it appears not.  I really hope we don't have to deploy the 100+Mb full PostGres component each time.

Reply:
Hmm, I thought the dependencies installed from the requirements.txt should be enough - I thought it was just the psycopg2 library that was needed. It could be that psycopg2-binary is needed instead - I'll make a note to investigate next week

Reply:
After a quick 5 sec look - the https://pypi.org/project/psycopg2-binary/ page says "You can also obtain a stand-alone package, not requiring a compiler or external libraries, by installing the psycopg2-binary package from PyPI" - so that should solve it. Unfortunately there's no easy way to install new packages into the deployed standalone Python installation (well there is, but it's messy) so I'll do a new release next week.